.envrc.local

```diff
@@ -0,0 +1,3 @@
+export AWS_DEFAULT_REGION=us-west-2
+
+export TT_SKIP_DESTROY=1
```

Comment:
Please add `.envrc.local` to the `.gitignore` file and then remove this file from the repo.